examples/src/main/java/com/mapbox/navigation/examples/activity/SimpleMapboxNavigationKt.kt

```diff
@@ -116,6 +117,25 @@ class SimpleMapboxNavigationKt : AppCompatActivity(), OnMapReadyCallback {
         }
     }
 
+    private val locationObserver = object : LocationObserver {
+        override fun onRawLocationChanged(rawLocation: Location) {
+            Timber.e("raw location %s", rawLocation.toString())
```

Comment:
I don't think that we should log it as `.e`

Reply:
Yea, this is just an example. I can update it.